File: claude/aspnet/rawdog/Models/Rating.cs
namespace LooseNotes.Models;

public class Rating
{
    public int Id { get; set; }
    public int Score { get; set; }
    public string Comment { get; set; } = string.Empty;
    public string SubmitterEmail { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

    public int NoteId { get; set; }
    public Note? Note { get; set; }
}

File: copilot/jsp/rawdog/loose-notes/src/main/java/com/loosenotes/model/Rating.java
package com.loosenotes.model;

public class Rating {
    private int id;
    private int noteId;
    private int userId;
    private int rating;
    private String comment;
    private String createdAt;
    private String username; // populated by JOIN

    public Rating() {}

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }

    public int getNoteId() { return noteId; }
    public void setNoteId(int noteId) { this.noteId = noteId; }

    public int getUserId() { return userId; }
    public void setUserId(int userId) { this.userId = userId; }

    public int getRating() { return rating; }
    public void setRating(int rating) { this.rating = rating; }

    public String getComment() { return comment; }
    public void setComment(String comment) { this.comment = comment; }

    public String getCreatedAt() { return createdAt; }
    public void setCreatedAt(String createdAt) { this.createdAt = createdAt; }

    public String getUsername() { return username; }
    public void setUsername(String username) { this.username = username; }
}
